src/SwedbankPay.Sdk/Utils.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("SwedbankPay.Sdk.Tests")]
+namespace SwedbankPay.Sdk
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+    using RestSharp;
+
+    internal static class Utils
```

Comment:
Please delete this class. We should not need to have any `Utils` in the code base. By using proper object oriented programming, all methods within this class can be moved into base classes and specific classes where they belong.